rllib/BUILD

```diff
@@ -1095,6 +1095,55 @@ py_test(
 #    args = ["--as-test", "--num-learners=2", "--num-gpus-per-learner=1"]
 # )
 
+# IQL
+# Pendulum-v1 (enormous)
```

Comment:
large or enormous? :)

We shouldn't allow enormous tests on the CI. See my comment below on maybe making this a nightly release test instead (see DreamerV3 Pendulum).

Reply:
This is not the test size but the data size. The `large` data shows less quality than the enormous one ;)